src/ipaperftest/core/plugin.py

```diff
@@ -223,24 +278,22 @@ def configure_server(self, ctx):
             }
             self.run_ansible_playbook_from_template(ANSIBLE_SERVER_ADD_REPO_PLAYBOOK,
                                                     "add_custom_repo", args, ctx)
+        etc_hosts = ("\n" + " " * 10).join(
+            [f"{ip} {host}.{self.domain} {host}" for host, ip in self.provider.hosts.items()])
```

Comment:
nit: is the short hostname needed in /etc/hosts?

Reply:
No, not really. I'll remove it.